operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -281,7 +281,7 @@
       "targets": [
         {
           "editorMode": "code",
-          "expr": "sum(increase(kube_pod_container_status_restarts_total{namespace=\"openshift-pipelines\", pod=~\"tekton-.*\"}[2m]))",
+          "expr": "sum(increase(kube_pod_container_status_restarts_total{namespace=\"openshift-pipelines\", pod=~\"tekton-.*\"}[$__range]))",
```

Comment:
I didn't think `$__range` worked in our non app SRE envs, but I have you change up now, and it is working

Assuming you copied the query as is from the app SRE panel, this is a happy find :-)

Reply:
Yes, I copied as it is.